### DISCUSSION TITLE
Working Async Storage package

### DISCUSSION THREAD

**Discussion by bartlomein** [callstack/react-native-visionos#109] [General]
Has anyone had to use something akin to Async Storage yet? If so, any packages that worked for you?

**Comment by bartlomein** [comment]
After messing around with a few packages [rn-secure-storage](https://github.com/talut/rn-secure-storage) worked with a small modification to the `podspec` file. 
[PR to get it working](https://github.com/talut/rn-secure-storage/pull/83)

**Comment by okwasniewski** [comment]
Looks like @thiagobrez got you covered: https://github.com/react-native-async-storage/async-storage/pull/1063 - new version of this lib has been released 🙌

**Reply by bartlomein** [reply]
Sweet! thanks @thiagobrez!
